tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -43,16 +43,16 @@ static const double EPS = 1e-3;
 
 TEST(TfGeometry, Frame)
 {
-  geometry_msgs::PoseStamped v1;
+  geometry_msgs::msg::PoseStamped v1;
   v1.pose.position.x = 1;
   v1.pose.position.y = 2;
   v1.pose.position.z = 3;
   v1.pose.orientation.x = 1;
-  v1.header.stamp = builtin_interfaces::msg::Time(2);
+  v1.header.stamp = tf2::TimePoint(2);
```

Comment:
Is this correct?  I thought that msg.header.stamp took builtin_interfaces time, not tf2::TimePoint.